tests/unit/multi.tcl

```diff
@@ -602,17 +602,14 @@ start_server {tags {"multi"}} {
     test {MULTI propagation of SCRIPT LOAD} {
         set repl [attach_to_replication_stream]
 
-        # make sure that SCRIPT LOAD inside MULTI is propagated in a transaction
+        # make sure that SCRIPT LOAD inside MULTI isn't propagated
         r multi
         r script load {redis.call('set', KEYS[1], 'foo')}
         set res [r exec]
         set sha [lindex $res 0]
 
         assert_replication_stream $repl {
-            {select *}
-            {multi}
-            {script load *}
-            {exec}
+            {}
```

Comment:
this assert may not get to fulfill its purpose.
let's add one additional command either inside the multi, or after it, and validate that (i.e that there's no SCRIPT LOAD before it.